masakari-controller/controller/masakari_controller.py

```diff
@@ -541,7 +541,12 @@ def _create_notification_list_db(self, jsonData):
                     msg = "Duplicate notifications. id:" + jsonData.get("id")
                     self.rc_util.syslogout(msg, syslog.LOG_INFO)
                     self.rc_util.syslogout(jsonData, syslog.LOG_INFO)
-
+            # Node is up
+            elif jsonData.get("type") == "rscGroup" and \
```

Comment:
Sachin, do we need a _check_repeated_notify here similar to the host-down case? or is this on purpose to retry enabling service on failure?